whistle/helpers.py

```diff
@@ -4,8 +4,8 @@
 from whistle import settings as whistle_settings
 
 
-def notify(request, recipient, event, actor=None, object=None, target=None):
-    NoticeManager.notify(request, recipient, event, actor, object, target)
+def notify(request, recipient, event, actor=None, object=None, target=None, details=None):
```

Comment:
"details" is a text field. None is not a valid value. Change it to empty string as default:

```suggestion
def notify(request, recipient, event, actor=None, object=None, target=None, details=''):
```